data/human/news.txt

```diff
@@ -1690,6 +1804,9 @@ news "republic soldier"
 			`We recently went to full alert, because somebody screamed "It's scrambled!" Turns out some rookie in the galley was enthusiastic about making scrambled eggs.`
 			"Curiosity killed the cat. Yes, the R.N.S. Curiosity landed right on top of the poor creature."
 			"We arrested a fortune teller yesterday; they didn't see it coming."
+			"Pirates were entering the ship port-side. So I took an EVA suit, flew around both ships and captured their ship."
+			"So we were fighting close quarters, when suddenly that Sparrow fired its lasers on us. We're lucky to still be alive."
+			"I once arrested one of those drunk flyers, but they walked free within two hours. The paperwork took me much longer. Next time I'll just blow them to pieces."
```

Comment:
```suggestion
			"I once arrested one of those drunk flyers, but they walked free within two hours. Very frustrating, because the paperwork took me much longer."
```